service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -41,6 +41,27 @@
 import com.ge.predix.acs.privilege.management.dao.SubjectEntity;
 import com.ge.predix.acs.rest.PolicyEvaluationResult;
 
+enum EntityType {
+    RESOURCE {
```

Comment:
can we please rewrite this enum using options listed here: https://stackoverflow.com/questions/6667243/using-enum-values-as-string-literals
I am not liking repetition of toString for each of the possible enum valuess